supports/integration-test/test_copy_1MB_5W.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from util import *
+
+
+FILE_SIZE = 1 * 1024 * 1024
+
+# add rule like 'file : path matches "/test/1MB/*" | sync -dest hdfs://namenode_ip:hdfs_port/test'
```

Comment:
Actually, you can submit rule with restful api, then no need for external actions.